### PR TITLE
Update cuda-python dependency to 11.7.1 from conda-forge

### PR DESCRIPTION
**Pull request by shwina**

### CODE REVIEW

**Review by galipremsagar — COMMENTED** [review]
Will need a version change here too: ~~https://github.com/rapidsai/integration/blob/branch-22.12/conda/recipes/versions.yaml#L48~~
https://github.com/rapidsai/integration/blob/branch-22.10/conda/recipes/versions.yaml#L47

**Comment by ajschmidt8** [comment]
> Will need a version change here too: https://github.com/rapidsai/integration/blob/branch-22.12/conda/recipes/versions.yaml#L48

Once this PR is merged, I'll merge `22.10` into `22.12` to ensure this change is propagated forward and to keep our `git` history in-sync.

**Comment by galipremsagar** [comment]
> > Will need a version change here too: https://github.com/rapidsai/integration/blob/branch-22.12/conda/recipes/versions.yaml#L48
> 
> Once this PR is merged, I'll merge `22.10` into `22.12` to ensure this change is propagated forward and to keep our `git` history in-sync.

Oh, Apologies. I mean't we will need to bump it in `22.10` only

**Review by ajschmidt8 — APPROVED** [review]

File: conda/recipes/versions.yaml [hunk 1]
```diff
@@ -45,7 +45,7 @@ cmakelang_version:
 cmake_setuptools_version:
   - '>=0.1.3'
 cuda_python_version:
-  - '>=11.5,<11.7.1'
+  - '>=11.7.1,<12.0"
```
Comment (galipremsagar) [inline conda/recipes/versions.yaml:48]:
```suggestion
  - '>=11.7.1,<12.0'
```

**Review by galipremsagar — APPROVED** [review]

**Comment by bdice** [comment]
This PR may need this change, too: https://github.com/shwina/integration/pull/1

**Pull request merged**